### PR TITLE
Add instruction details to MVP builtins

### PR DESCRIPTION
**Pull request by dcodeIO**
Like instruction info for SIMD in https://github.com/AssemblyScript/website/pull/76, but for MVP builtins.

### CODE REVIEW

File: src/stdlib/builtins.md [hunk 1]
```diff
@@ -147,77 +147,182 @@ The following generic built-ins compile to WebAssembly instructions directly.
...
 * ```ts
   function copysign<T>(x: T , y: T): T
   ```
-  Composes a 32-bit or 64-bit float from the magnitude of `x` and the sign of `y`.
+  <details><summary>Composes a 32-bit or 64-bit float from the magnitude of <code>x</code> and the sign of <code>y</code>.</summary>
+
+  | T   | Instruction
+  |-----|-------------
+  | f32 | f32.copysign
+  | f64 | f64.copysign
```
Comment (MaxGraey) [inline src/stdlib/builtins.md:276]: I guess make sense add this:
```ts
| i8, u8, i16, u16, i32, u32, i64, u64, bool | *omitted*
```
for consistency

File: src/stdlib/builtins.md [hunk 2]
```diff
@@ -147,77 +147,182 @@ The following generic built-ins compile to WebAssembly instructions directly.
...
 * ```ts
   function sqrt<T>(value: T): T
   ```
-  Calculates the square root of a 32-bit or 64-bit float.
+  <details><summary>Calculates the square root of a 32-bit or 64-bit float.</summary>
+
+  | T   | Instruction
+  |-----|-------------
+  | f32 | f32.sqrt
+  | f64 | f64.sqrt
```
Comment (MaxGraey) [inline src/stdlib/builtins.md:312]: the same

File: src/stdlib/builtins.md [hunk 3]
```diff
@@ -226,12 +331,37 @@ Similarly, the following built-ins emit WebAssembly instructions accessing or ot
...
+
+  | T        | Instruction  | If context is i64
+  |----------|--------------|-------------------
+  | i8       | i32.load8_s  | i64.load8_s
+  | u8       | i32.load8_u  | i64.load8_u
+  | i16      | i32.load16_s | i64.load16_s
+  | u16      | i32.load16_u | i64.load16_u
+  | i32      | i32.load     | i64.load32_s
+  | u32      | i32.load     | i64.load32_u
+  | i64, u64 | i64.load     |
```
Comment (MaxGraey) [inline src/stdlib/builtins.md]:
```suggestion
  | i64, u64 |              |  i64.load
```

File: src/stdlib/builtins.md (hunk 1 above)
Comment (dcodeIO) [reply src/stdlib/builtins.md:276]: This is what the compiler does, there are no fillers for integers here. Would you prefer to have fillers here?

Comment (dcodeIO) [reply src/stdlib/builtins.md:276]: For context, `omitted` above means that the operation accepts the type argument, but does not emit an instruction and just pipes through, which sometimes makes sense for integers. But not here.

Comment (MaxGraey) [reply src/stdlib/builtins.md:276]: Ah wait, isn't `*omitted*` means unsupported (compile error)?

File: src/stdlib/builtins.md (hunk 2 above)
Comment (dcodeIO) [reply src/stdlib/builtins.md:312]: Similar to the one above, in that we do not polyfill an integer version here.

File: src/stdlib/builtins.md (hunk 3 above)
Comment (dcodeIO) [reply src/stdlib/builtins.md]: If the type argument is `i64` or `u64`, it always emits an `i64.load`, independent of whether context is `i64`. Could add a `*same*` here, or `*n/a*`, wdyt?

Comment (MaxGraey) [reply src/stdlib/builtins.md]: `*n/a*` will be fine. Same for `i64.store` case

**Review by MaxGraey — APPROVED** [review]

**Comment by dcodeIO** [comment]
Renamed `*omitted*` to `*none*` to indicate that no instruction is emitted, and named the blanks `*n/a*`.

**Pull request merged**